### PR TITLE
Avoid importlib-metadata conflict with tox

### PR DESCRIPTION
**Pull request by greg-hellings**
importlib-metadata does not support installing versions after 2.0 on
Python versions prior to py38.
Also fix the path for ANSIBLE_CONFIG file

#### PR Type

- Bugfix Pull Request

### CODE REVIEW

**Comment by greg-hellings** [comment]
Getting tox/build-containers to build properly is being a pain. build-containers tests all the main plugins, including molecule-azure, which currently fails because ansible[azure] is no longer a thing.

File: tox.ini [hunk 1]
```diff
@@ -38,7 +38,7 @@ passenv =
     SSH_AUTH_SOCK
     TERM
 setenv =
-    ANSIBLE_CONFIG={toxinidir}/dev/null
+    ANSIBLE_CONFIG={toxinidir}/.ansible.cfg
```
Comment (ssbarnea) [inline tox.ini:41]: This this really needed, why? Seems unrelated.

Comment (greg-hellings) [reply tox.ini:41]: If ANSIBLE_CONFIG points to a non-existent path, then the value is ignored and Ansible falls back to any that exist on the local system. This causes local runs of tests to fail, at least on my system, because of some of my ~/.ansible.cfg settings.

True, this isn't strictly necessary, but it was necessary to pass in my environment.

**Review by ssbarnea — APPROVED** [review]
